Examples/Algorithms/TrackFindingExaTrkX/include/ActsExamples/TrackFindingExaTrkX/PrototracksToParsAndSeeds.hpp

```diff
@@ -0,0 +1,84 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/EventData/ProtoTrack.hpp"
+#include "ActsExamples/EventData/SimSeed.hpp"
+#include "ActsExamples/EventData/Track.hpp"
+#include "ActsExamples/Framework/DataHandle.hpp"
+#include "ActsExamples/Framework/IAlgorithm.hpp"
+
+namespace ActsExamples {
+
+class PrototracksToParsAndSeeds final : public IAlgorithm {
```

Comment:
not too happy with the name but I don't have a great alternative. maybe `PrototracksToParametersAndSeeds`

nitpick: we mix "Prototracks" and "ProtoTracks"

Reply:
Maybe only `PrototrackToParameters`? Because the seeds are really just a byproduct here...